Gemfile

```diff
@@ -9,7 +10,7 @@ gem 'faraday', '0.9.1'
 gem 'faraday_middleware', '0.9.1'
 gem 'figaro', '1.1.1'
 gem 'hiredis', '0.6.0'
-gem 'health-check', '1.1.1', require: 'rack/health_check'
+#gem 'health-check', '1.1.1', require: 'rack/health_check'
```

Comment:
you can remove this line, since we are not using them, we can always add them back if needed in the specific project